themes/classic/_dev/js/components/product-select.js

```diff
@@ -32,7 +32,7 @@ export default class ProductSelect {
     let $arrows =   $('.js-modal-arrows');
     let $thumbnails = $('.js-modal-product-images');
     let $onsale =  $('.on-sale');
-    let $new = $('.new');
```

Comment:
I didn't say you need to remove it. just to use it. 
It's not consistent to have $pack and no $new, or $new and no $pack. You must use both variables or not.

Reply:
I wasn't using it.
Why was $new wasn't create before ?
I'll create it for the one which already exist.

Reply:
I created it